Contributing.md

```diff
@@ -10,15 +10,15 @@ If you're reading this, you're probably creating a Pull Request or planning to d
 
 2. Clone the forked repository.
 
-   ```bash
```

Comment:
You were not supposed to remove the `triple backticks`. It is used to create `code-blocks ` which leads to better readability.

Reply:
Do this at every place which requires code-blocks , you can preview the markdown file for checking if the code block is created